src/main/java/duke/UI.java

```diff
@@ -54,22 +55,26 @@ public static String added(String taskType) {
 
     /**
      * Returns a message when a task is deleted from TaskList
+     *
      * @param taskNumber Number of task in TaskList
      */
     public static String delete(int taskNumber) {
         return String.format("Got it! I've removed task %d.", taskNumber);
     }
 
     /**
-     * Prints a message to show the number of current tasks
+     * Returns a message to show the number of current tasks
+     *
      * @param numberOfTasks Number of current tasks
      */
     public static String numberOfTasks(int numberOfTasks) {
         return String.format("You currently have %d tasks.%n", numberOfTasks);
     }
 
     /**
-     * Lists the current Tasks in taskArrayList with numbering
+     * Returns a string of the current Tasks in taskArrayList with numbering
+     *
+     * @return String of tasks
      */
     public static String list(TaskList taskList) {
         String list = "Here are your current tasks:\n";
```

Comment:
Is it better to make this String a constant value? which is naming it as LIST and static final?

Reply:
I don't think I can do that because after that there is some string concatenation. Good spot though! I actually haven't been using any constant names because I can't find where to use them.